File: backend/models/product.js
const sequelize = require("../db");
const { DataTypes } = require("sequelize");
const Product = sequelize.define(
  "Product",
  {
    id: {
      primaryKey: true,
      allowNull: false,
      autoIncrement: true,
      type: DataTypes.INTEGER,
    },
    productName: {
      type: DataTypes.STRING,
      allowNull: true,
    },
    productNo: {
      type: DataTypes.INTEGER,
      allowNull: true,
    },
    serialNo: {
      type: DataTypes.INTEGER,
      allowNull: true,
    },
    lotNo: {
      type: DataTypes.STRING,
      allowNull: true,
    },
    productionDate: {
      type: DataTypes.DATE,
      allowNull: true,
    },
    expirationDate: {
      type: DataTypes.DATE,
      allowNull: true,
    },
  },
  { timestamps: false }
);

Product.sync({ alter: true });
console.log("==> The table for the Product model was just (re)created!");

module.exports = Product;

File: Med Storage App/Entities/Product.cs
namespace Med_Storage_App.Entities
{
    public class Product
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        public int No { get; set; }
        public int SerialNo { get; set; }
        public string? LotNo { get; set; }
        public int Quantity { get; set; }
        public DateTime ProductionDate { get; set; }
        public DateTime ExpirationDate { get; set; }
    }
}
